nilearn/input_data/nifti_labels_masker.py

```diff
@@ -142,6 +148,18 @@ def __init__(self, labels_img, background_label=0, mask_img=None,
         self.memory_level = memory_level
         self.verbose = verbose
 
+        available_reduction_strategies = {'mean', 'median',
+                                          'minimum', 'maximum',
+                                          'standard_deviation', 'variance'}
+
+        if strategy in available_reduction_strategies:
+            self.strategy = strategy
+        else:
+            raise ValueError(str.format("Strategy '{}' not in {}",
+                                        strategy,
+                                        available_reduction_strategies))
```

Comment:
```python
"Invalid strategy '{}'. Valid strategies are {}".format(strategy, available_reduction_strategies)
```

Reply:
Fixed. Also refactored to remove the else clauses.